### DISCUSSION TITLE
Support periods (`.`) in field names

### DISCUSSION THREAD

**Discussion by bamboowonder** [directus/directus#12686] [Feature Requests]
### Describe the Bug

A postgres column that includes a period does not display correctly in the main table view, or the configuration screens.
In the picture, note that the 4 columns after "Recruiter" all contain periods.
(as seen in the column picker on upper right)
In second picture, all of the property panes are blank (Schema, Field, Interface... etc)
In the third picture, detail page, the field names do display correctly.

![Screenshot from 2022-04-08 22-41-38](https://user-images.githubusercontent.com/15368570/162558418-af32c749-ca7a-48a5-87bf-bea5fda8945d.png)

![Screenshot from 2022-04-08 22-51-09](https://user-images.githubusercontent.com/15368570/162558593-ef1db4ee-8227-4129-a729-2a4befd3bda9.png)

![Screenshot from 2022-04-08 22-54-06](https://user-images.githubusercontent.com/15368570/162558657-e83ddae6-8143-4109-bbe7-968c801f3b06.png)

### To Reproduce

create any column that includes a period (.) and then try to display it in a table

**Comment by rijkvanzanten** [comment]
Moving this to a feature request discussion, as Directus currently doesn't support `.` in field names

**Comment by rijkvanzanten** [comment]
Directus currently doesn't support using period characters in field names. This is due to the fact that we rely on "dot notation" for various operations in the platform (like grouping translations `t('fields.my_custom_field_name')`, or query params `?sort=author.first_name`).

**Reply by bamboowonder** [reply]
Totally agree, periods in column names is dodgy anyway.   Unfortunately it is a table I acquired from another dev.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
